package.json

```diff
@@ -3254,6 +3254,7 @@
     "open": "^6.3.0",
     "parinfer": "^3.12.0",
     "posthtml-parser": "^0.11.0",
+    "react-dom": "^18.2.0",
```

Comment:
I don't know if we need React for something in particular. If we do, then of course we should use it. Otherwise, it may be better to use something with less bloat, like Dumdom, or maybe Replicant.

Reply:
It's for reagent, which I consider very lightweight and should be used lightly for this. If you think there's any performance issue with it that would be significant for our use-case, we should discuss that, but switching it out later should be pretty simple.

Reply:
I'm open to alternatives if there's a good reason to use something else.

Reply:
Dumdom looks interesting. Using a library that supports react gives us reach into the react ecosystem though, which could be nice in the future for more rich features and interactivity (think not recreating the wheel so much).

Reply:
My experience with React is that it changes all the time in breaking ways, making it high maintenance. With Dumdom we would avoid this, as the vdom library it uses is super tiny and stable. With Replicant we would get a pure ClojureScript vdom as well, and have even more of the familiar Clojure stability (though I am not sure if the Replicant API is marked as stable yet).

However, I am not opposed to React if we have a good reason to use it.

Reply:
I say we move forward with it for a first-release, and maybe be conscious of our usage of it (keeping it light), and if we come across issues we can switch it out later.

I'll continue to keep other options in mind and think about the React usage for now...

Reply:
> I say we move forward with it for a first-release

I may reconsider this. I need to look at Dumdom more closely.

Reply:
Having had a closer look at Replicant, In think it's better to use that instead of Dumdom. They are quite similar, but Replicant drops the dependency on Snabbdom, as well as some legacy API requirements that Dumdom carries.